server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts

```diff
@@ -1,9 +1,23 @@
 import { Router, Request, Response } from 'express';
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { GET_EXPOSURE_INFO } from '../../DBService/Exposure/Query';
+import { UPDATE_EXPOSURE, CREATE_EXPOSURE } from '../../DBService/Exposure/Mutation';
 
-const exosureRoute = Router();
+const exposureRoute = Router();
 
-exosureRoute.get('/', (request: Request, response: Response) => {
-    response.send('Hello from Exposure route');
+exposureRoute.get('/:investigationId', (request: Request, response: Response) => {
+    graphqlRequest(GET_EXPOSURE_INFO, request.headers, {investigationId : parseInt(request.params.investigationId)})
+    .then((result: any) => response.send(result));
 })
 
-export default exosureRoute;
\ No newline at end of file
+exposureRoute.post('/', (request: Request, response: Response) => {
+        graphqlRequest(CREATE_EXPOSURE, request.headers, { data: request.body.data })
+        .then((result: any) => response.send(result));
+})
+
+exposureRoute.post('/update', (request: Request, response: Response) => {
```

Comment:
I think you can use PUT instead of POST and then delete the /update in the request